### ISSUE TITLE
Blur issue happened on the flatten region 

### ISSUE THREAD

**Issue by tommy830725** [cswry/OSEDiff#33]
Hi author, thanks for your great work. I try to reproduce the training flow as the paper. However I face a problem on my retrained model. Even if the sharpen and detail region on inference results looks make sense, the flatten or low frequency region is blurred. The state of blur is more serious than the input low quality patterns. Is the root cause of this problem due to the normalized l1 (or l2 loss), should add some gradient normalizing methods on the Lora training or others? Sorry, I try lots of methods but still not work. Hope you can provide some help, thank you.

**Comment by romantic-black** [comment]
> Hi author, thanks for your great work. I try to reproduce the training flow as the paper. However I face a problem on my retrained model. Even if the sharpen and detail region on inference results looks make sense, the flatten or low frequency region is blurred. The state of blur is more serious than the input low quality patterns. Is the root cause of this problem due to the normalized l1 (or l2 loss), should add some gradient normalizing methods on the Lora training or others? Sorry, I try lots of methods but still not work. Hope you can provide some help, thank you.

Hello, is the blur problem solved after removing L2 loss? I also encountered a similar problem.
